### PR TITLE
Update the links to our code of conduct

### LINKED ISSUE #87: [BUG] Issue template links to CODE OF CONDUCT of rubygems repo
**Branch/Commit:**

Main branch

**Expected behavior:**

I should see a link to the CODE_OF_CONDUCT.md file inside this repo https://github.com/fastruby/dash/blob/main/CODE_OF_CONDUCT.md

**Actual behavior:**

CoC link points to rubygems https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md

**I will abide by the [code of conduct] (https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)**

### PR DESCRIPTION
**Pull request by cleicar**
This closes #87.

I will abide by the code of conduct.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ If the issue has an effect in the frontend, include any relevant screenshots and
 
 **Logs**
 
-Include relevant log snippets or files here. 
+Include relevant log snippets or files here.
 
-**I will abide by the [code of conduct] (https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)**
+**I will abide by the [code of conduct] (https://github.com/fastruby/dash/blob/main/CODE_OF_CONDUCT.md)**
```
Comment (arielj) [inline .github/ISSUE_TEMPLATE/bug_report.md:64]: I think we can do `**I will abide by the [code of conduct](./CODE_OF_CONDUCT.md)**` so it's easier to maintain. The link href works relative to the current file's directory https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-readmes#relative-links-and-image-paths-in-readme-files

File: README.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Effort has been taken to ensure that private information is excluded from the re
 
   ## Code of Conduct
 
-  Everyone interacting in the Dash project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](CODE_OF_CONDUCT.md).
+  Everyone interacting in the Dash project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/fastruby/dash/blob/main/CODE_OF_CONDUCT.md).
```
Comment (arielj) [inline README.md:95]: similar here in the README, but without the `./` since it's in the same folder

**Review by arielj — CHANGES_REQUESTED** [review]
Hey @cleicar I added one comment to make it easier to maintain in case it ever changes name/org/branch, using relative paths instead

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-readmes#relative-links-and-image-paths-in-readme-files

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/ombulabs/dash/tree/master.svg?style=svg&circle-token=aa41e55b03a167988f14667d78d1d7c0183f2656)](https://circleci.com/gh/ombulabs/dash/tree/master)  [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)
+[![CircleCI](https://circleci.com/gh/ombulabs/dash/tree/master.svg?style=svg&circle-token=aa41e55b03a167988f14667d78d1d7c0183f2656)](https://circleci.com/gh/ombulabs/dash/tree/master)  [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](https://github.com/fastruby/dash/blob/main/CODE_OF_CONDUCT.md)
```
Comment (arielj) [inline README.md:1]: similar here in the README, but without the `./` since it's in the same folder

File: README.md (hunk 2 above)
Comment (arielj) [inline README.md:95]: this was ok without the absolute url

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (arielj) [reply .github/ISSUE_TEMPLATE/bug_report.md:64]: also, I think there's a space between `[code of conduct]` and the `(...)` breaking the link formatting (it was not introduced on this PR, it happens on main too, but maybe it's worth to fix it here)

Comment (arielj) [reply .github/ISSUE_TEMPLATE/bug_report.md:64]: UPDATE on this: just tested and the way to use relative paths on a PR is `[code of conduct](../blob/main/CODE_OF_CONDUCT.md)`

I guess we still need the `main` part on the link

**Comment by cleicar** [comment]
Thanks for your feedback @arielj, make sense!

But I'm finding hard to test it here and make sure that it works.

**Review by arielj — APPROVED** [review]
You can test the PR template change by adding that link here on this PR for the PR template

Also, I used that link version on this issue https://github.com/fastruby/dash/issues/108 and the link works for issue templates too.

It looks good to me

**Pull request merged**